js/state/frameStateUtil.js

```diff
@@ -84,7 +84,16 @@ export function findIndexForFrameKey (frames, key) {
  * Obtains the frameProps index in the frames
  */
 export function getFramePropsIndex (frames, frameProps) {
-  return frames.findIndex(found => found.get('key') === frameProps.get('key'))
+  return frames.findIndex(found => {
+    try {
+      return found.get('key') === frameProps.get('key')
+    } catch (e) {
+      // todo
+      // catch '.get is not a function'
+      // this seems to happen when testing
+      // possibly dispatchToIPC related
```

Comment:
Sounds like you have `found` which is not an Immutable object.  Example:

{a: 3} vs Immutable.fromJS({a: 3})